lms/djangoapps/verify_student/management/commands/retry_failed_photo_verifications.py

```diff
@@ -40,12 +51,63 @@ def add_arguments(self, parser):
             help='Use arguments from the SSPVerificationRetryConfig model instead of the command line.',
         )
 
+        parser.add_argument(
+            '--status',
+            action='store',
+            dest='status',
+            default='must_retry',
+            type=str,
+            help='SoftwareSecurePhotoVerifications status to filter for'
+        )
+
+        parser.add_argument(
+            '--start_datetime',
+            action='store',
+            dest='start_datetime',
+            default=None,
+            type=str,
+            help='Start date for a date range of SoftwareSecurePhotoVerifications; '
+                 'Should be formatted as YYYY-mm-dd HH:MM:SS; '
+                 'Requires the end_datetime arg.',
+        )
+
+        parser.add_argument(
+            '--end_datetime',
+            action='store',
+            dest='end_datetime',
+            # "YYYY-mm-dd HH:MM:SS"
+            default=None,
+            type=str,
+            help='End date for a date range of SoftwareSecurePhotoVerifications; '
+                 'Should be formatted as YYYY-mm-dd HH:MM:SS; '
+                 'Requires the start_datetime arg.',
+        )
+
+        parser.add_argument(
+            '--batch_size',
+            action='store',
+            dest='batch_size',
+            type=int,
+            default=300,
+            help='Maximum number of SoftwareSecurePhotoVerifications to process. '
+                 'This helps avoid overloading the database while updating large amount of data.'
+        )
+
+        parser.add_argument(
+            '--sleep_time',
+            action='store',
+            dest='sleep_time',
+            type=int,
+            default=10,
+            help='Sleep time in seconds between update of batches'
+        )
+
     def get_args_from_database(self):
         """ Returns an options dictionary from the current SSPVerificationRetryConfig model. """
 
         sspv_retry_config = SSPVerificationRetryConfig.current()
         if not sspv_retry_config.enabled:
-            log.warning('SSPVerificationRetryConfig is disabled or empty, but --args-from-database was requested.')
+            log.error('SSPVerificationRetryConfig is disabled or empty, but --args-from-database was requested.')
```

Comment:
If the sspv_retry_config is empty, then no args such as `status` or `start_datetime` are passed in, causing the test to fail due to KeyErrors from those args not existing. As such, I changed this to an error since it halts any execution of the command.